docs/dev/policy_documents/merging-and-promoting-conventions.md

```diff
@@ -0,0 +1,40 @@
+# Merging and Promoting Conventions
+
+## Summary
+[summary]: #summary
+
+This policy defines a 'fast-track' merge and promote policy for a subset of core-plans.  
+
+## Motivation
+[motivation]: #motivation
+
+If someone who contributes heavily to core plans, has a pull request for a minor change open for 7days +, with all tests passing, it would be good if the same person can also merge and promote if there is a lack of peer reviews. One contributor (Graham Wilson, predominant) has hundreds of updates in his backlog, but has been holding back because of delays in the review process.
+
+## Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+What plan is eligable for the fast-track criteria:
+
+* Only identified 'veteran' core-plan contributor's have permission to use this fast-track policy.  This list will be agreed and specified in the MAINTAINERS.md
+* This fasttrack will only apply to non-baseplan PRs and PRs that do not have a label 'DO-NOT-MERGE' or are otherwise marked for caution.
```

Comment:
```suggestion
* This fast-track will only apply to non-base plan PRs and PRs that do not have a label 'DO-NOT-MERGE' or are otherwise marked for caution.
```